### PR TITLE
[CONTENT] More Thoughts for Traits

### PR DESCRIPTION
**Pull request by tobes03**
## About The Pull Request

Just padding the general trait-locked thoughts.

## Why This Is Good For ClanGen

Adds some flavour :)

## Proof of Testing

(childish)
image
image
image

(strict)
image

(compassionate)
image

(adventurous)
image

### CODE REVIEW

File: resources/dicts/thoughts/alive/general.json [hunk 1]
```diff
@@ -753,8 +794,11 @@
     {
         "id": "careful_to_med",
         "thoughts": [
-            "Is getting a small scratch checked out by r_c",
-            "Is urging a Clanmate to get a scratch checked out"
+            "Is getting a small scratch checked out by the medicine cat",
+            "Is urging a Clanmate to get an scratch checked out",
```
Comment (scribblecrumb) [inline resources/dicts/thoughts/alive/general.json]:
```suggestion
            "Is urging a Clanmate to get a scratch checked out",
```

File: resources/dicts/thoughts/alive/general.json [hunk 2]
```diff
@@ -422,8 +428,10 @@
     {
         "id": "adventurous_gen_to_alive_kits",
         "thoughts": [
-            "Involves the kits in a game of make-believe about traveling through a strange land",
-            "Encourages the kits to explore as much as possible"
+            "Involves the kits in a game of make believe about traveling through a strange land",
```
Comment (arisaema12) [inline resources/dicts/thoughts/alive/general.json]:
```suggestion
            "Involves the kits in a game of make-believe about traveling through a strange land",
```
'make believe' is a phrase (eg i make believe that...) while 'make-believe' is a noun ('unicorns are make-believe').
sorry for the english lesson lol, i wasn't sure how to explain

File: resources/dicts/thoughts/alive/general.json [hunk 3]
```diff
@@ -493,7 +501,13 @@
             "Is feeling proud of {PRONOUN/m_c/self}",
             "Is daydreaming about a Clan celebration in {PRONOUN/m_c/poss} honor",
             "Offers to help {PRONOUN/m_c/poss} Clanmates with their duties",
-            "Imagines how {PRONOUN/m_c/subject} would run the Clan better"
+            "Wonders how {PRONOUN/m_c/subject} would run the Clan better",
+            "Considers what {PRONOUN/m_c/poss} first actions as leader will be",
```
Comment (arisaema12) [inline resources/dicts/thoughts/alive/general.json]:
```suggestion
            "Considers what {PRONOUN/m_c/poss} first actions as leader would be",
```
maybe change this to 'would' so it doesn't sound like m_c is _definitely_ going to be leader?

File: resources/dicts/thoughts/alive/general.json [hunk 4]
```diff
@@ -521,7 +535,10 @@
         "id": "ambitious_to_leader",
         "thoughts": [
             "Is asking the Clan leader what {PRONOUN/m_c/subject} can do to help out around camp",
-            "Has been imitating the Clan leader's behavior recently"
+            "Has been imitating the Clan leader's behaviour recently",
+            "Knows {PRONOUN/m_c/subject}{VERB/m_c/'re/'s} a shoo-in for deputy",
+            "Wonders if the Clan leader talks about {PRONOUN/m_c/object} when {PRONOUN/m_c/subject}{VERB/m_c/'re/'s} not around",
+            "Tells the Clan leader {PRONOUN/lead_name/subject}{VERB/lead_name/'ve/'s} been doing a wonderful job leading"
```
Comment (arisaema12) [inline resources/dicts/thoughts/alive/general.json]:
```suggestion
            "Tells lead_name that {PRONOUN/lead_name/subject}{VERB/lead_name/'ve/'s} been doing a wonderful job leading"
```
for readability, having 'leader' and 'leading' in the same sentence just feels wrong imo

File: resources/dicts/thoughts/alive/general.json [hunk 5]
```diff
@@ -542,7 +559,12 @@
             "Is lecturing about something someone said wrong offpawedly",
             "Is certain {PRONOUN/m_c/subject} {VERB/m_c/look/looks} better than any other cat in the Clan",
             "Took the lead during a patrol even though nobody wanted {PRONOUN/m_c/object} to",
-            "Is bragging to everyone"
+            "Has simply decided not to do what {PRONOUN/m_c/subject} {VERB/m_c/were/was} told",
+            "Is bragging to everyone",
+            "Looks smug despite not doing anything impressive",
+            "Grooms in an extra ostentatious way",
+            "Knows {PRONOUN/m_c/subject}{VERB/m_c/'re/'s} the cleverest in the Clan",
```
Comment (arisaema12) [inline resources/dicts/thoughts/alive/general.json]: maybe change 'knows' to 'thinks'? 'knows' implies to me that they _are_ the cleverest cat in the clan

File: resources/dicts/thoughts/alive/general.json [hunk 6]
```diff
@@ -569,7 +591,11 @@
             "Imagines how {PRONOUN/m_c/subject} would run the Clan better",
             "Licks fresh-kill blood off {PRONOUN/m_c/poss} lips a little too intently",
             "Has been glaring at a certain cat in camp with an odd look in {PRONOUN/m_c/poss} eye",
-            "Daydreams about slaughtering every cat {PRONOUN/m_c/subject} {VERB/m_c/despise/despises}"
+            "Daydreams about slaughtering every cat {PRONOUN/m_c/subject} {VERB/m_c/despise/despises}",
+            "Shreds {PRONOUN/m_c/poss} bedding absentmindedly",
```
Comment (arisaema12) [inline resources/dicts/thoughts/alive/general.json:595]:
```suggestion
            "Shreds {PRONOUN/m_c/poss} bedding absent-mindedly",
```

File: resources/dicts/thoughts/alive/general.json (hunk 1 above)
Comment (arisaema12) [inline resources/dicts/thoughts/alive/general.json]:
```suggestion
            "Is urging a Clanmate to get a scratch checked out",
```

File: resources/dicts/thoughts/alive/general.json [hunk 7]
```diff
@@ -790,12 +834,53 @@
             "Forgot where {PRONOUN/m_c/subject} put something, and every other cat offers to help {PRONOUN/m_c/object} find it",
             "Is a little exhausted from how many cats come to {PRONOUN/m_c/object} for things",
             "Feels happy knowing {PRONOUN/m_c/subject}{VERB/m_c/'re/'s} someone others can talk to",
-            "Carefully curates {PRONOUN/m_c/poss} appearance to be dazzling, but not too dazzling"
+            "Carefully curates {PRONOUN/m_c/poss} appearance to be dazzling, but not too dazzling",
+            "Is just sooooo popular, lately",
+            "Helps a shy Clanmate get closer with one of {PRONOUN/m_c/poss} friends",
+            "Counts {PRONOUN/m_c/poss} friends on {PRONOUN/m_c/poss} toebeans",
```
Comment (arisaema12) [inline resources/dicts/thoughts/alive/general.json]: i feel like 'doesn't have enough toebeans to count all their friends!' would be better for a charismatic cat, while this thought feels more for a loving cat ('count your blessings' kinda thing). that's purely my opinion tho

File: resources/dicts/thoughts/alive/general.json [hunk 8]
```diff
@@ -1440,6 +1626,23 @@
             "living"
         ]
     },
+    {
+        "id": "loyal_to_elder",
```
Comment (scribblecrumb) [inline resources/dicts/thoughts/alive/general.json:1635]: this one and the previous block seem to have had their constraints flipped around a bit.  This block should have the elder constraint, not the previous.

File: resources/dicts/thoughts/alive/general.json [hunk 9]
```diff
@@ -1784,13 +2022,48 @@
...
+            "Sneezes eleven times in a row",
+            "Wonders if StarClan ever gets bored",
+            "Makes a poultice of mud and grass"
         ],
         "main_trait_constraint": [
             "strange"
         ]
     },
+    {
+        "id": "sneaky_gen",
```
Comment (scribblecrumb) [inline resources/dicts/thoughts/alive/general.json:2021]: this block seems to have been duplicated from an edited block above?

**Review by scribblecrumb — APPROVED** [review]

**Review by scribblecrumb — CHANGES_REQUESTED** [review]
bleh hit the wrong button, meant to request changes, not approve xD

**Comment by tobes03** [comment]
wow there were some wacky errors in there. i guess a lot of the errors arose from the time it spent sitting in my drafts lol...

File: resources/dicts/thoughts/alive/general.json [hunk 10]
```diff
@@ -1025,35 +1130,48 @@
     {
         "id": "compassionate_warrior_to_living_med_cat",
```
Comment (arisaema12) [inline resources/dicts/thoughts/alive/general.json]:
```suggestion
        "id": "compassionate_to_living_med_cat",
```

File: resources/dicts/thoughts/alive/general.json [hunk 11]
```diff
@@ -1025,35 +1130,48 @@
...
         "id": "competitive_gen",
         "thoughts": [
-            "Is challenging cats around camp to participate in some friendly competition",
-            "Challenged a Clanmate to a staring contest",
-            "Challenged a Clanmate to a staring contest... and won",
-            "Challenged a Clanmate to a staring contest... and lost",
-            "Has been a good sport about losing to a Clanmate",
-            "Has been a bit of a bad sport about losing to a Clanmate",
-            "Challenged a Clanmate to a race",
+            "Is pestering other cats to join in challenges around camp",
```
Comment (arisaema12) [inline resources/dicts/thoughts/alive/general.json]:
```suggestion
            "Is pestering other cats to join in with challenges around camp",
```

File: resources/dicts/thoughts/alive/general.json [hunk 12]
```diff
@@ -1092,20 +1215,20 @@
             "apprentice",
             "warrior",
             "deputy",
-            "leader"
-        ],
-        "main_trait_constraint": [
-            "confident"
-        ]
```
Comment (arisaema12) [inline resources/dicts/thoughts/alive/general.json:1099]: it looks like you’ve deleted the trait constraint here?

File: resources/dicts/thoughts/alive/general.json [hunk 13]
```diff
@@ -1092,20 +1215,20 @@
...
     },
     {
         "id": "cunning_gen",
         "thoughts": [
             "Convinces a Clanmate to do something {PRONOUN/m_c/subject} could easily have done {PRONOUN/m_c/self}",
             "Is always five steps ahead of {PRONOUN/m_c/poss} Clanmates",
             "Knows exactly how to get {PRONOUN/m_c/self} out of a mess {PRONOUN/m_c/subject} orchestrated",
-            "Grooms {PRONOUN/m_c/poss} pelt to <i>just</i> this side of perfection",
-            "Got a Clanmate to do {PRONOUN/m_c/poss} work for {PRONOUN/m_c/object}"
+            "Carefully curates {PRONOUN/m_c/poss} appearance to be the most dazzling in the Clan",
```
Comment (arisaema12) [inline resources/dicts/thoughts/alive/general.json:1229]: imo the original ‘grooms their pelt to _just_ this side of perfection’ fits well for a cunning cat, maybe we could keep both thoughts?

File: resources/dicts/thoughts/alive/general.json [hunk 14]
```diff
@@ -1175,8 +1309,11 @@
             "Thinks that {PRONOUN/m_c/subject} could take on a dog solo",
             "Hopes to earn more battle scars to show off",
             "Wishes {PRONOUN/m_c/subject} had more scars",
-            "Pounces on a Clanmate unexpectedly, meowing about how they should be ready at any moment",
-            "Tears up {PRONOUN/m_c/poss} fresh-kill far too violently for others' liking"
+            "Pounced on a Clanmate unexpectedly, meowing about how they should be ready at any moment",
+            "Tears up {PRONOUN/m_c/poss} fresh-kill far too violently for others' liking",
+            "Knows {PRONOUN/m_c/subject}{VERB/m_c/'re/'s} a force to be reckoned with",
+            "Wonders if a war will break out anytime soon",
+            "Is ready to defend c_n to the death"
```
Comment (arisaema12) [inline resources/dicts/thoughts/alive/general.json:1320]: this wording feels a little odd to me, maybe rephrase it as ‘would fight to the death for c_n’ or ‘is ready to defend c_n with their life’

File: resources/dicts/thoughts/alive/general.json [hunk 15]
```diff
@@ -1248,7 +1413,15 @@
             "Was scared by an owl last night and is too embarrassed to tell anyone",
             "Wonders if {PRONOUN/m_c/poss} fur is unkempt and whether others can notice",
             "Worries that {PRONOUN/m_c/subject} {VERB/m_c/aren't/isn't} contributing enough",
-            "Thinks about all the mistakes {PRONOUN/m_c/subject}{VERB/m_c/'ve/'s} made"
+            "Thinks about all the mistakes {PRONOUN/m_c/subject}{VERB/m_c/'ve/'s} made",
+            "Dreads the worst case scenario",
```
Comment (arisaema12) [inline resources/dicts/thoughts/alive/general.json]:
```suggestion
            "Dreads the worst-case scenario",
```

File: resources/dicts/thoughts/alive/general.json [hunk 16]
```diff
@@ -1248,7 +1413,15 @@
...
+            "Is too nervous to tell another cat they're wrong",
+            "Wonders if r_c has ever struggled with feelings of inadequacy",
+            "Knows {PRONOUN/m_c/subject}{VERB/m_c/'re/'s} doing the best {PRONOUN/m_c/subject} can, but feels guilty anyway",
+            "Apologizes for something that wasn't {PRONOUN/m_c/subject} fault",
+            "Fakes it 'til {PRONOUN/m_c/subject} {VERB/m_c/make/makes it}",
+            "Boasts loudly about all {PRONOUN/m_c/poss} accomplishments",
```
Comment (arisaema12) [inline resources/dicts/thoughts/alive/general.json:1427]: this thought feels like it fits a confident cat, not an insecure one

File: resources/dicts/thoughts/alive/general.json [hunk 17]
```diff
@@ -1408,36 +1594,53 @@
...
+        "main_trait_constraint": [
+            "loyal"
+        ],
         "main_status_constraint": [
             "apprentice",
-            "medicine cat apprentice",
-            "mediator apprentice",
             "warrior",
             "medicine cat",
-            "mediator",
```
Comment (arisaema12) [inline resources/dicts/thoughts/alive/general.json:1429]: were mediator & mediator apprentice supposed to have been removed here?

File: resources/dicts/thoughts/alive/general.json [hunk 18]
```diff
@@ -1460,7 +1663,13 @@
             "Counts every cat in the Clan to make sure nobody's missing",
             "Stumbles over {PRONOUN/m_c/poss} words, never quite able to get it right",
             "Keeps {PRONOUN/m_c/poss} eyes on the skies, afraid an eagle could come get {PRONOUN/m_c/object}",
-            "Tries to breathe deeply to no avail"
+            "Tries to breathe deeply to no avail",
+            "Thinks every little tickle on {PRONOUN/m_c/poss} fur is a spider",
+            "Wonders if it's possible to die from a thorn in {PRONOUN/m_c/poss} paw",
+            "Sneaks out of {PRONOUN/m_c/poss} nest at moon-high to make sure the camp's okay",
```
Comment (arisaema12) [inline resources/dicts/thoughts/alive/general.json]:
```suggestion
            "Sneaks out of {PRONOUN/m_c/poss} nest at moonhigh to make sure the camp's okay",
```

File: resources/dicts/thoughts/alive/general.json [hunk 19]
```diff
@@ -1504,7 +1713,13 @@
             "Is running after a colorful beetle",
             "Is letting out mrrows of laughter",
             "Has been playing tricks on {PRONOUN/m_c/poss} fellow Clanmates",
-            "Is making up songs and encouraging others to join in"
+            "Is making up songs and encouraging others to join in",
+            "Tries to organize a Clan-wide game of tag",
+            "Gives fully-grown Clanmates badger rides... or tries",
+            "Runs around camp at moon-high, getting out all {PRONOUN/m_c/poss} energy",
```
Comment (arisaema12) [inline resources/dicts/thoughts/alive/general.json]:
```suggestion
            "Runs around camp at moonhigh, getting out all {PRONOUN/m_c/poss} energy",
```

File: resources/dicts/thoughts/alive/general.json [hunk 20]
```diff
@@ -1856,12 +2108,29 @@
         "id": "thoughtful_gen",
         "thoughts": [
             "Is offering helpful advice to a sad Clanmate",
-            "Prepares the perfect gift for a friend",
-            "Notices a Clanmate is tired and offers to take their chores",
-            "Gets an injured Clanmate a piece of fresh-kill"
+            "Prepared the perfect gift for a friend",
+            "Noticed a Clanmate was tired and offered to take on their chores",
+            "Gets an injured Clanmate a piece of fresh-kill",
+            "Reassures an apprentice who's nervous to go on patrol",
```
Comment (arisaema12) [inline resources/dicts/thoughts/alive/general.json]: this thought can be deleted - it’s already in thoughtful_to_warrior_apprentice

File: resources/dicts/thoughts/alive/general.json [hunk 21]
```diff
@@ -1998,7 +2280,15 @@
...
             "Sits quietly, observing the Clan dynamics with a knowing gaze",
             "Predicts the weather by observing the sky and the behavior of prey",
-            "Considers multiple outcomes before making a decision"
+            "Considers multiple outcomes before making a decision",
+            "Knows time will heal all wounds",
+            "Purrs softly at a Clanmate taking a rash course of action",
+            "Gives r_c some good advice",
+            "De-escalates an argument before it even really starts",
+            "Defuses the tension with a joke at {PRONOUN/m_c/poss} own expense",
+            "Asks an older cat for advice",
```
Comment (arisaema12) [inline resources/dicts/thoughts/alive/general.json]: might want to add an age constraint to this one in a separate block

File: resources/dicts/thoughts/alive/general.json [hunk 22]
```diff
@@ -720,7 +755,13 @@
             "Is patching up a hole in the camp wall",
             "Is helping to reinforce the walls of the elders' den",
             "Fortifies the camp walls for the fourth time this week",
-            "Feels at peace"
+            "Feels at peace",
+            "Wonders if the camp is protected enough",
+            "Double checks that {PRONOUN/m_c/poss} joke didn't upset r_c",
```
Comment (arisaema12) [inline resources/dicts/thoughts/alive/general.json]:
```suggestion
            "Double-checks that {PRONOUN/m_c/poss} joke didn't upset r_c",
```

File: resources/dicts/thoughts/alive/general.json (hunk 6 above)
Comment (tobes03) [reply resources/dicts/thoughts/alive/general.json:595]: The adverb is actually most commonly unhyphenated according to M-W

File: resources/dicts/thoughts/alive/general.json (hunk 7 above)
Comment (tobes03) [reply resources/dicts/thoughts/alive/general.json]: I see what you mean! The former meaning is what I was going for, so I'll modify it

File: resources/dicts/thoughts/alive/general.json (hunk 11 above)
Comment (tobes03) [reply resources/dicts/thoughts/alive/general.json]: Yeah I think this one was an existing thought that was already phrased weird; I'm gonna edit it further for clarity.

File: resources/dicts/thoughts/alive/general.json (hunk 13 above)
Comment (tobes03) [reply resources/dicts/thoughts/alive/general.json:1229]: Didn't mean the delete that one, whoops. I was clumsy with resolving merge conflicts, as the other mystery deletions show...

**Comment by tobes03** [comment]
Okay I snuck in more edits changing the construction of a lot of verbs. There's no clear consensus on whether thoughts should generally be in the "Verbs" (present) tense or the "Is Verbing" (present continuous). There are some where is makes sense for one over the other, but very often, the present continuous is applied when it just makes the sentence longer and clunky, which is particularly bad for a feature that relies on economy of words. But I might open another branch to edit the thoughts more fully.

File: resources/dicts/thoughts/alive/general.json [hunk 23]
```diff
@@ -2924,7 +3221,7 @@
     {
         "id": "apprentices_to_mentor",
         "thoughts": [
-            "Is making {PRONOUN/m_c/poss} mentor laugh",
+            "Mkaes {PRONOUN/m_c/poss} mentor laugh",
```
Comment (arisaema12) [inline resources/dicts/thoughts/alive/general.json]:
```suggestion
            "Makes {PRONOUN/m_c/poss} mentor laugh",
```

File: resources/dicts/thoughts/alive/general.json [hunk 24]
```diff
@@ -538,7 +538,7 @@
             "Has been imitating the Clan leader's behaviour recently",
             "Knows {PRONOUN/m_c/subject}{VERB/m_c/'re/'s} a shoo-in for deputy",
             "Wonders if the Clan leader talks about {PRONOUN/m_c/object} when {PRONOUN/m_c/subject}{VERB/m_c/'re/'s} not around",
-            "Tells the Clan leader {PRONOUN/lead_name/subject}{VERB/lead_name/'ve/'s} been doing a wonderful job leading"
+            "Tells lead_name {PRONOUN/lead_name/subject}{VERB/lead_name/'ve/'s} been doing a wonderful job leading"
```
Comment (scribblecrumb) [inline resources/dicts/thoughts/alive/general.json]: not sure why we're utilizing lead_name here when the r_c cat is already constrained to being the leader?  It would be more consistent, imo, if we just used r_c.
```suggestion
            "Tells r_c {PRONOUN/r_c/subject}{VERB/r_c/'ve/'s} been doing a wonderful job leading"
```

**Review by scribblecrumb — CHANGES_REQUESTED** [review]

**Review by scribblecrumb — APPROVED** [review]
